pkg/client/cache/fifo.go

```diff
@@ -241,6 +241,26 @@ func (f *FIFO) Replace(list []interface{}, resourceVersion string) error {
 	return nil
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *FIFO) Resync() error {
+	f.lock.Lock()
+	defer f.lock.Unlock()
+
+	inQueue := make(map[string]bool)
```

Comment:
nit: could probably make this with `len(f.queue)`, and the value doesn't really need to be `bool` -- `struct{}` works just as well.